Packs/Base/Scripts/DBotFindSimilarIncidents/DBotFindSimilarIncidents.yml

```diff
@@ -0,0 +1,180 @@
+args:
+- default: false
+  description: Incident id to get prediction, if empty - using the current incident
+    id.
+  isArray: false
+  name: incidentId
+  required: false
+  secret: false
+- default: false
+  description: 'Fields with short or long text that will be taken into account in
+    order to compute similarity - Example: commandline, URL...'
+  isArray: false
+  name: similarTextField
+  required: false
+  secret: false
+- default: false
+  description: Fields for which exact match is not necessary but will boost the score.
+  isArray: false
+  name: similarCategoricalField
+  required: false
+  secret: false
+- default: false
+  description: Additional incident fields that will be displayed.
+  isArray: false
+  name: fieldsToDisplay
+  required: false
+  secret: false
+- default: false
+  description: Fields for which exact match is required - help to make the query to
```

Comment:
Demisto = xsoar, and I wouldn't specify the reason for this field, not sure if relevant for the user